www/install/php/Update-19.04.5.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/*
+ * Copyright 2005 - 2019 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+include_once __DIR__ . "/../../class/centreonLog.class.php";
+$centreonLog = new CentreonLog();
+
+/**
+ * Update session duration value to the max allowed : 120 min
+ */
+try {
+    $stmt = $pearDB->query(
+        'SELECT `value` FROM `options` WHERE `key` = "session_expire"'
+    );
+    $sessionValue = $stmt->fetch();
+
+    if ($sessionValue > 120) {
```

Comment:
Could you reuse the _SESSION_DURATION_LIMIT_ constant in some way ?

Reply:
I can't, as the value won't be available until the restart of the services after the upgrade.
As the value was set to 7200 sec in the 50-centreon.ini, this value will be the default one after the process is acheived.
Then the customer will be able to modify it, and the form will take in account this new value, as the max value.